cli/js/lib.deno.ns.d.ts

```diff
@@ -2411,9 +2482,19 @@ declare namespace Deno {
    *
    * Finally, a number of output options are also available.
    *
-   *      const out = Deno.inspect(obj, {showHidden: true, depth: 4, colors: true, indentLevel: 2});
+   *      const out = Deno.inspect(obj, {
+   *        showHidden: true,
+   *        depth: 4,
+   *        colors: true,
+   *        indentLevel: 2
+   *      });
+   *
+   * Warning: The exact form of the string output is under consideration and may
+   * change.
    *
    */
+  // TODO(ry) None of the options work currently, remove InspectOptions from
+  // API.
```

Comment:
This situation is more complicated: 
it turns out that `console.log()` uses `stringifyArgs` function underneath that supports `depth` and `indentLevel` whilst `Deno.inspect()` uses `stringify` function which supports only `depth`.

Reply:
Shouldn't output from both `console.log` and `Deno.inspect` be identical with a difference that `console.log` logs message to console while `Deno.inpect` returns string?